src/App.js

```diff
@@ -29,28 +31,30 @@ class App extends Component {
 
   render() {
     return (
-      <MuiThemeProvider theme={theme}>
-        <NavigtaionContext.Provider
-          value={{ source: 'ALL', date: getNewDate() }}
-        >
-          <div className="App">
-            <div className="globe-container">
-              <div className="col-lg-12 col-sm-12 main-section">
-                <MainSection showGlobe={this.state.showGlobe} />
-              </div>
-              <div className="m-6">
-                <div className="guide-1">
-                  <GroupedButton
-                    showGlobe={this.state.showGlobe}
-                    toggleDisplay={this.toggleDisplay}
-                  />
+      <MuiPickersUtilsProvider utils={DateFnsUtils}>
+        <MuiThemeProvider theme={theme}>
+          <NavigtaionContext.Provider
```

Comment:
`NavigationContext` is already provided in the top level, I don't think we need to define it here again

Reply:
Resolved!